pkg/eventmonitor/consumers/testutil/testutil.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2024-present Datadog, Inc.
 
-//go:build (linux || windows) && test
```

Comment:
why removing the windows tag? this util is not used to test on windows? (the eventMonitor platform supports windows )

Reply:
The testutil it's based on is only enabled for linux, adding the windows tag causes a build failure.